astropy/units/tests/test_equivalencies.py

```diff
@@ -681,8 +681,10 @@ def test_temperature():
 def test_temperature_energy():
     x = 1000 * u.K
     y = (x * constants.k_B).to(u.keV)
-    assert_allclose(x.to_value(u.keV, u.temperature_energy()), y.value)
-    assert_allclose(y.to_value(u.K, u.temperature_energy()), x.value)
+    assert_allclose(
```

Comment:
This one is very weird. Why did it get worse?

Reply:
My theory: https://github.com/astropy/astropy/pull/8761#discussion_r287566612

Reply:
Ah, I think I found it: you need to also update the defaults in `constants.si` and `constants.cgs`! May need to check whether this affects any of the other tests...

Reply:
Thanks, @mhvk ! I'll update those files as well and maybe back-out some of the test changes in case they are no longer necessary.